scripts/run-clang-format.py

```diff
@@ -30,7 +30,6 @@
 except ImportError:
```

Comment:
Please don't format this file. This is an external library, so we should keep the formatting the same for comparability / upgrade reasons.